to.etc.domui/src/main/resources/resources/themes/scss/winter/_buttonBar.scss

```diff
@@ -11,6 +11,9 @@
 .ui-bb-middle .ui-button {
 	margin: 5px 10px 5px 0;
 }
+.ui-bb-middle .ui-button:first-of-type {
+	margin-left: 10px;
```

Comment:
What exactly is solved by this? Can you show a screenshot? The button bar is used really a lot so changes are a bit dangerous.

Reply:
well, if we have more width in buttons then in content then first button gets sticked to left border of dialog, I had that reported as issue actually.

Reply:
re-fixed to be used only within floating div style